### ISSUE TITLE
ArgumentError - Unmatched double quote

### ISSUE THREAD

**Issue by jadeirving9** [CocoaPods/CocoaPods#9352]
* [x] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

Run `pod install`

## What did you expect to happen?

Install all pod dependencies correctly.

### Error

```
ArgumentError - Unmatched double quote: 
...
Searching for inspections failed: undefined method `map' for nil:NilClass

## CocoaPods Environment

### Stack

```
   CocoaPods : 1.8.4
        Ruby : ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-darwin17]
    RubyGems : 2.7.2
        Host : Mac OS X 10.15.1 (19B88)
       Xcode : 11.2.1 (11B500)
         Git : git version 2.10.1
Ruby lib dir : /usr/local/Cellar/ruby/2.4.2_1/lib
Repositories : trunk - CDN - https://cdn.cocoapods.org/
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.1.0
cocoapods-trunk       : 1.4.1
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
platform :ios, '9.0'
require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'

target 'Tulip' do
  # Pods for Tulip
  pod 'FBLazyVector', :path => "../node_modules/react-native/Libraries/FBLazyVector"
  pod 'FBReactNativeSpec', :path => "../node_modules/react-native/Libraries/FBReactNativeSpec"
  pod 'RCTRequired', :path => "../node_modules/react-native/Libraries/RCTRequired"
  pod 'RCTTypeSafety', :path => "../node_modules/react-native/Libraries/TypeSafety"
  pod 'React', :path => '../node_modules/react-native/'
  pod 'React-Core', :path => '../node_modules/react-native/'
  pod 'React-CoreModules', :path => '../node_modules/react-native/React/CoreModules'
  pod 'React-Core/DevSupport', :path => '../node_modules/react-native/'
  pod 'React-RCTActionSheet', :path => '../node_modules/react-native/Libraries/ActionSheetIOS'
  pod 'React-RCTAnimation', :path => '../node_modules/react-native/Libraries/NativeAnimation'
  pod 'React-RCTBlob', :path => '../node_modules/react-native/Libraries/Blob'
  pod 'React-RCTImage', :path => '../node_modules/react-native/Libraries/Image'
  pod 'React-RCTLinking', :path => '../node_modules/react-native/Libraries/LinkingIOS'
  pod 'React-RCTNetwork', :path => '../node_modules/react-native/Libraries/Network'
  pod 'React-RCTSettings', :path => '../node_modules/react-native/Libraries/Settings'
  pod 'React-RCTText', :path => '../node_modules/react-native/Libraries/Text'
  pod 'React-RCTVibration', :path => '../node_modules/react-native/Libraries/Vibration'
  pod 'React-Core/RCTWebSocket', :path => '../node_modules/react-native/'

  pod 'React-cxxreact', :path => '../node_modules/react-native/ReactCommon/cxxreact'
  pod 'React-jsi', :path => '../node_modules/react-native/ReactCommon/jsi'
  pod 'React-jsiexecutor', :path => '../node_modules/react-native/ReactCommon/jsiexecutor'
  pod 'React-jsinspector', :path => '../node_modules/react-native/ReactCommon/jsinspector'
  pod 'ReactCommon/jscallinvoker', :path => "../node_modules/react-native/ReactCommon"
  pod 'ReactCommon/turbomodule/core', :path => "../node_modules/react-native/ReactCommon"
  pod 'Yoga', :path => '../node_modules/react-native/ReactCommon/yoga'

  pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
  pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
  pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'




  target 'TulipTests' do
    inherit! :search_paths
    # Pods for testing
  end

  use_native_modules!
end

target 'Tulip-tvOS' do
  # Pods for Tulip-tvOS

  target 'Tulip-tvOSTests' do
    inherit! :search_paths
    # Pods for testing
  end

end
```

**Comment by dnkoutso** [comment]
This is not a CocoaPods issue currently. It seems you are having an unmatched `"` in a podspec or your `Podfile`.

Need a sample app that clearly demonstrates the issue.

**Comment by jadeirving9** [comment]
Hi sorry I can't provide a sample (code for work) but I can tell you the steps to possibly recreate the issue.

I am working on a react-native project, and tried to add realm to the project via node with the command:

 npm install realm --save

then I linked realm with the project:

react-native link realm

and finally I wanted to do:

 cd ios && pod install && cd .. 

but the error I entered above is produced. I have looked through the pod file and replaced all double quotes with single quotes, but that has done nothing. 
 
FYI I only posted this as a cocoapods issue because of the error pod install gave me - it recommended I post it here.
